libs/product/src/reducers/composite-product-entities/composite-product-entity.ts

```diff
@@ -1,7 +1,10 @@
 import { Dictionary } from '@ngrx/entity';
+
+import { ID } from '@daffodil/core';
+
 import { DaffCompositeConfigurationItem } from '../../models/composite-configuration-item';
 
 export interface DaffCompositeProductEntity {
-	id: string;
+	id: ID;
```

Comment:
I think this should be `DaffProduct['id']`

Reply:
changed